contents/tutorials/api-capture-events.mdx

```diff
@@ -258,15 +417,315 @@ body = {
 
 response = requests.post(url, headers=headers, json=body)
 
+print(response.json())
+```
+```js file=NodeJS
+// Load environment variables from .env file
+require('dotenv').config();
+
+const headers = {
+    "Content-Type": "application/json",
+};
+
+const body = {
+    "api_key": "<ph_project_api_key>",
+    "properties": {
+        "distinct_id": "test-user-javascript",
+        "alias": "test-user-javascript-alias"
+    },
+    "event": "$create_alias"
+};
+
+const url = "<ph_client_api_host>/i/v0/e/";
+
+fetch(url, {
+    method: 'POST',
+    headers: headers,
+    body: JSON.stringify(body)
+})
+.then(response => response.json())
+.then(data => console.log(data))
+.catch(error => console.error('Error:', error));
+```
+</MultiLanguage>
+
+## Capturing errors
+
+Everything is an event in PostHog. [Error tracking](/docs/error-tracking) is no different. You can manually capture errors by sending an `$exception` event with the following properties:
+
+| Property | Description |
+|----------|-------------|
+| `$exception_list` | A list of exception objects with detailed information about each error. Each exception can include a `type`, `value`, `mechanism`, `module`, and a `stacktrace` with `frames` and `type`. You can find the expected schema as types for both [exception](https://github.com/PostHog/posthog/blob/master/rust/cymbal/src/types/mod.rs#L39) and [stack frames](https://github.com/PostHog/posthog/blob/master/rust/cymbal/src/langs/custom.rs#L12) in our Rust repo |
+| `$exception_fingerprint` | (Optional) The identifier used to group issues. If not set, a unique hash based on the exception pattern will be generated during ingestion |
+
+Here's an example of how to capture an error:
+
+<MultiLanguage>
+
+```bash
+#!/bin/bash
+
+# Load environment variables from .env file
```

Comment:
Why do we need to load all the environment variables?

Reply:
Also, do we need to escape everything? The other examples don't escape things and they work fine

Reply:
When I run this, I get `zsh: command not found: md5sum
{"status":"Ok"}% `